pony-kafka/customnet/custom_tcp_connection_handler.pony

```diff
@@ -165,10 +167,10 @@ class CustomTCPConnectionHandler is TCPConnectionHandler
     end
     _connected = true
     ifdef linux then
-      AsioEvent.set_writeable(_event, true)
+      AsioEventHelper.set_writeable(_event, true)
```

Comment:
The whole `AsioEventHelper` (and prior to that `AsioEvent`) concept was dropped when this enhancement to ponyc was upstreamed. This needs to be updated to do what TCPConnection does in upstream ponyc.

Reply:
Ok.  I made the switch on Wallaroo after talking to Sean.  I'll do this tomorrow.